Use single quote as apostrophe (#3395)
theforeman/foreman-documentation: guides/common/modules/snip_prerequisites-upgrading-project-server.adoc

.{SmartProxy} considerations

ifdef::katello,orcharhino,satellite[]

> * If you use content views to control updates to a {SmartProxyServer}'s base operating system, or for {SmartProxyServer} repository, you must publish updated versions of those content views.

Reviewer: ```suggestion
* If you use content views to control updates to the base operating system of {SmartProxyServer}, or for {SmartProxyServer} repository, you must publish updated versions of those content views.
```

Author: Applied.
